client/src/modules/users/user.modal.html

```diff
@@ -21,7 +21,10 @@
       <div>
         <input name="display_name" bh-unique="/users/{{UserModalCtrl.user.display_name}}" ng-model="UserModalCtrl.user.display_name" autocomplete="off" ng-maxlength="UserModalCtrl.maxLength" class="form-control" required>
 
-        <span ng-show="UserForm.display_name.$error.unique" class="glyphicon glyphicon-remove form-control-feedback" id="unique-error-icon"></span>
+        <div ng-show="UserForm.display_name.$error.unique" translate class="text-danger">
```

Comment:
From your screenshot, looks like you need to make this a `<span>` and have [class="help-block"](https://getbootstrap.com/docs/3.3/css/#forms-help-text) on it, to make it behave the same as the other help blocks.